examples/pylab_examples/image_origin.py

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env python
 """
 You can specify whether images should be plotted with the array origin
 x[0,0] in the upper left or upper right by using the origin parameter.
 You can also control the default be setting image.origin in your
 matplotlibrc file; see http://matplotlib.org/matplotlibrc
 """
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 
-x = arange(100.0)
+x = np.arange(100.0)
 x.shape = (10, 10)
 
 interp = 'bilinear'
-#interp = 'nearest'
+# interp = 'nearest'
```

Comment:
your autopep8 is showing again... it is not supposed to be putting a space in here for commented out code. This is true in a few places.